tests/plugins/tabletools/columndeletion.js

```diff
@@ -0,0 +1,74 @@
+/* bender-tags: editor,unit */
```

Comment:
Tests connected with `tableselection` plugin should be inserted into `tableselection/integrations` directory. It allows to easily find all issues related to integrations between `tableselection` and other plugins.